### PR TITLE
Add h1 tag to html and styles to h1 tag

### PR DESCRIPTION
**Pull request by asifaminisonline**
created new git repo and cloned the HTML and CSS folder in it after that I created HTML, CSS, and test t.md file and push the code to GitHub

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+    color: green;
+}
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ ] As stated in the project requirement your .css file should includes one class with styles for your header (make it in your favorite color), Kindly update it.
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @asifaminisonline  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻


✔️ You used correct GitHub Flow.
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ]  I recommend that you fix the linters errors in your styles.css. To have the project approved, the linter result should be green. AKA: Linter should report NO errors.
- [ ] Your  README file should be descriptive all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo
- [ ] As stated in the project requirement A test.md file with the text "This file should be ignored by git" is actually ignored by git and is not present in your GitHub repository and also you don't have .gitignore file listing test.md file and node_modules directory. 

Capture

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```
Comment (Nemwel-Boniface) [inline style.css]: - [ ] As requested by the previous reviewer, the CSS file is supposed to include a CSS class selector styling the h1 element. This involves having the h1 with a class attribute and styling it using the class selector and not the element selector.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @asifaminisonline     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] I am currently still able to see the test.md file in your Github, which is not the desired output. To effectively ignore it, delete the file, commit the changes, then re-add it again and then commit the changes again. The test.md file will effectively be ignored. The idea behind is to commit the .gitignore before the actual file that you want to ignore.

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (Owusu-Desmond) [inline style.css]: - [ ] As requested by the previous reviewer, the CSS file is supposed to include a CSS class selector styling the h1 element. This involves having the h1 with a class attribute and styling it using the class selector and not the element selector. For more information on how to do that please visit [using a class selector to style elements](w3schools.com/cssref/sel_class.php)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @asifaminisonline  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented
:heavy_check_mark: Good PR description.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asifaminisonline** [comment]
please explain it in detail what are you saying

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada:  :heavy_check_mark: 

Hi @asifaminisonline, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**kindly check some optional comments I left under the review and work on them**_ :pray: 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark:



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [OPTIONAL] I would like to recommend that you should always try to give your pull request a descriptive titile and summary. A title like `Hello Microverse Set Up Project` would have been better to `Add h1 tag to html and styles to h1 tag`. In the description section, you're expected to state what you've worked on in the project. Take the format below as an example

```
  In this pull request, I was able to:
  -   set up the linters
  -   create index file and add `h1` tag with `Hello Microverse` as content.
  -   create css file to style the content of the index file.
  -   create test.md file and add it to `.gitignore` file.
  -  etc.
```
- Kindly modify both the title and the summary section of your project for future reference.

**Pull request merged**